brilift/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+mod rt;
+pub mod translator;
+
+use crate::translator::{find_func, Translator};
+use bril_rs::Program;
+use cranelift_object::ObjectModule;
+
+pub fn compile(program: &Program, target: Option<String>, opt_level: &str, output: &str) {
```

Comment:
It looks like `main.rs` now largely duplicates this function (for the `!args.jit` case). What do you think about turning that into a function call to this new `compile` function?